src/main/java/com/jalasoft/webservice/entitities/TextFile.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
+package com.jalasoft.webservice.entitities;
+
+import com.jalasoft.webservice.entitities.ModelFile;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/** TextFile : Class to manipulate information related to CSV files
+ * Version : 1.0
+ * Date: 9/19/2019
+ */
+
+public class TextFile extends ModelFile {
```

Comment:
Do we need a `TextFile` class? why don't move  `saveTextIntoFile` method to `FileManager` and give the `text`, `fullfilepath` Strings as arguments? like the following:
```java
public void saveTextIntoFile(String text, String fullfilepath) {
```

Reply:
done